pkg/metrics/metrics.go

```diff
@@ -109,6 +109,8 @@ func getGardenMetricsDefinitions() map[string]*prometheus.Desc {
 				"seed_iaas",
 				"seed_region",
 				"uid",
+				"has_errors",
```

Comment:
```suggestion
				"has_user_errors",
```
As suggested by @dguendisch